native-pkcs11-keychain/src/macos/key.rs

```diff
@@ -199,7 +192,7 @@ impl KeychainPublicKey {
         let der = sec_key.external_representation().ok_or("no external representation")?;
         let key_ty = sec_key_algorithm(&sec_key)?;
         Ok(Self {
-            public_key_hash: sec_key.application_label().ok_or("no application_label")?,
+            id: sec_key.application_label().ok_or("no application_label")?,
```

Comment:
Can we comment or link to bugs that describe why this is chosen as the ID. (Just to protect ourselves from the reasons being lost)